DuplicateAssemblyScanner/DuplicateAssemblyScanner/Util/Md5.cs

```diff
@@ -0,0 +1,56 @@
+namespace DuplicateAssemblyScanner.Util {
+    using System;
+    using System.IO;
+    using System.Security.Cryptography;
+
+    /// <summary>
+    /// Utility functions for files.
+    /// </summary>
+    public class Md5 {
+
+        /// <summary>
+        /// Tries to get the MD5 hash for <paramref name="fullPath"/>.
+        /// </summary>
+        /// 
+        /// <param name="fullPath">Full path of file, including file name.</param>
+        /// <param name="md5Hash">If successful, the calculated MD5 hasn for <paramref name="fullPath"/>.</param>
+        /// <param name="removeHyphens">If <c>true</c>, hyphens will be removed form the returned string.</param>
+        /// 
+        /// <returns>Returns <c>true</c> if successful, otherwise <c>false</c>.</returns>
+        internal static bool TryGetHash(string fullPath, out string md5Hash, bool removeHyphens = false) {
```

Comment:
`out` parameters should be placed at the end of the method parameter list; in this case, the `removeHyphens` parameters will become a required one. Create an overload without `removeHyphens` if you wish.

Reply:
The param was no longer needed so removed it.